angular_token_auth.js

```diff
@@ -13,6 +13,8 @@
         AUTH_HEADER_PREFIX: 'Token',
         ALLOWED_HOSTS: [],
         COOKIE_PATH: null
+        // Optional settings examples:
+        // STORAGE_METHOD: 'localstorage',
```

Comment:
Yeah, like why have you put `// STORAGE_METHOD: 'localstorage'`? The reason for putting that one example should also apply to all the other examples.